src/ipc/dialog.ts

```diff
@@ -0,0 +1,17 @@
+import {
+  IpcMain,
+  IpcMainInvokeEvent,
+  dialog,
+  OpenDialogOptions,
+} from 'electron';
+
+export const channel = 'dialog';
+
+export default function register(ipcMain: IpcMain): void {
+  ipcMain.handle(
+    channel,
+    async (event: IpcMainInvokeEvent, options: OpenDialogOptions) => {
+      return dialog.showOpenDialog(options);
+    },
+  );
+}
```

Comment:
When I first wrote this, I was only thinking of the "Open" dialog, whereas I could have built this in such a way that also gives us access to the "Save" dialog. Using the "Save" dialog for saving the screenshot may be cleaner than using the link generated method. 

I think we should either a) change this to enable more dialogs (through arguments or separate endpoints) or b) rename this one to be specific to the open dialog. 

I'm happy to do this.

Reply:
I like the idea of enabling more dialogs!